api/models/fixtures/cron_labels.yml

```diff
@@ -7,4 +7,4 @@ main:
 really-long-process-type-name:
   build: .
   labels:
-    - convox.cron.really-long-cron-job-name=0 * * * ? bin/myjob
```

Comment:
Just made the change to allow dashes in cronjob names for now.

Allowing dots in cronjob names doesn't seem like a good idea because we're using dots to signify namespaces.

I think there's a separate PR to be made to enforce starting and ending with an alphanumeric characters and disallowing consecutive dots and dashes across all of our labels.